server/src/main/java/com/google/sps/utils/Ad.java

```diff
@@ -0,0 +1,201 @@
+/* 
+ * Description: Creates ad objects 
+ * Author: Kira Toal
+ * Date: June 24, 2020
+ */
+package com.google.sps.utils;
+
+import java.lang.StringBuilder;
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.List; 
+
+public class Ad {
+
+    public String id; 
+    public String advertiser;  
+    public String startDate;
+    public String endDate;
+    public long impressionsMin; 
+    public long impressionsMax; 
+    public boolean ageTargetingEnabled; 
+    public List<String> genderTarget; 
+    public List<String> geoTarget;
+    public long spendMin; 
+    public long spendMax; 
+    public String headline; 
+    public String link;
+    public String content;  
+    public String headlineSentiment; 
+    public String headlineTerms;
+    public String contentSentiment; 
+    public String contentTerms; 
+
+  public Ad() {
```

Comment:
I think it's okay to write it like this: `public Ad() {}`, then put the comment right above the constructor

Reply:
Cool!